examples/simple/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/newrelic/opentelemetry-exporter-go/examples
```

Comment:
Should this be more like
```suggestion
module github.com/newrelic/opentelemetry-exporter-go/examples/simple
```
?

Reply:
Yes, thanks, I'll be putting in a gin example beside it soon.